configs/webpack/base.coffee

```diff
@@ -10,7 +10,7 @@ _ = require 'lodash'
 # base config, true for all builds no matter what conditions
 BASE_CONFIG =
   cache: true
-  devtool: 'source-map'
+  devtool: 'cheap-module-eval-source-map'
```

Comment:
yeah, this is only for dev mode.  Production sets it to `source-map` (line 136 below)